update audit page

haochizzle opened the pull request:
- remove card view
- add audit table
- add repo version

closes:[#29]

waymobetta commented on docs/07-audits/index.md:
```diff
@@ -6,9 +6,11 @@ description: The following details audit reports related to the various Sygma re
+| Date | Auditor | Audited Repos | Link | Status |
+| ---- |-------------| ------ | ------- | :----: |
+| Sept 6, 2022 | [KALOS (prev. HAECHI AUDIT)](https://www.kalos.xyz/) | [sygma-solidity v1.0](https://github.com/sygmaprotocol/sygma-solidity/releases/tag/v1.0.0), [sygma-relayer v1.0](https://github.com/sygmaprotocol/sygma-relayer/releases/tag/v1.0.0) | [Link](/assets/[HAECHI%20AUDIT]%20Sygma%20Audit%20Report%20v1.1.pdf) | ✅ |
+| May 18, 2023 | [Least Authority](https://leastauthority.com/) | [sygma-solidity v2.5](https://github.com/sygmaprotocol/sygma-solidity/releases/tag/v2.3.0), [sygma-relayer v1.5](https://github.com/sygmaprotocol/sygma-relayer/releases/tag/v1.5.0), [sygma-substrate-pallets v0.2](https://github.com/sygmaprotocol/sygma-substrate-pallets/releases/tag/sygma-bridge-v0.2.0)| [Link](https://docs.buildwit ...
```
Release versions inconsistent for sygma-solidity v2.5 and the tag which is for v2.3

haochizzle replied:
ah good catch, i was following @MakMuftic's versioning on PROD at [https://docs.buildwithsygma.com/audits/v1.5/](https://docs.buildwithsygma.com/audits/v1.5/)

haochizzle replied:
also we should scrap this because base branch is `develop`. i will open a separate PR to update @MakMuftic's versioning on PROD

waymobetta commented:
There's no need to scrap, you just change base

haochizzle commented:
@waymobetta it should be scrapped because `v2` is already up-to-date with the new table, updated versioning (2.3), and removed cards. I'll let ya close this PR once you review what's already in [v2](https://github.com/sygmaprotocol/docs/blob/v2/docs/07-audits/index.md).
